one_fm/overrides/leave_allocation.py

```diff
@@ -6,25 +6,25 @@
 class LeaveAllocationOverride(LeaveAllocation):
     def validate(self):
         super().validate()
-        self.validate_employee_gender_for_maternity()
+        self.validate_employee_gender()
         self.validate_hajj_leave()
     
-    def validate_employee_gender_for_maternity(self):

```

Comment:
You removed this aspect, hope this was intentional

Reply:
I didn't add all this by myself. I said it earlier that if I pull staging to my branch to resolved the conflict it might cause another discrepancy